filemanager/__init__.py

```diff
@@ -76,7 +76,7 @@ def rename_if_exists(self, folder, file):
                     full_path = folder + name + '.' + str(i) + extension
                     if not os.path.exists(full_path):
                         break
-                return name + '.' + str(i) + extension
+                return "{0}.{1}{2}".format(name, str(i), extension)
```

Comment:
Please use `'` instead of `"` for string literals, to be consistent with the rest of the code.

Reply:
gotcha. will do.